hw/ip/prim/dv/prim_present/crypto_dpi_present/boxes.inc

```diff
@@ -0,0 +1,44 @@
+/*
+ * S-Boxes and P-Boxes for
+ * Implementation of PRESENT in C
+ * v2.1, 10/13/2008
+ *
+ * Thomas Siebert, thomas.siebert@rub.de
```

Comment:
i think strictly speaking...these files are supposed to be vendor'd in if possible.
I'm not actually sure what the protocol is if they don't have a github page we can download from.

I don't personally really care though...

Reply:
yeah...I had that issue too since there's not a github page for this model.
Is there a better approach for integrating these files? I did something very similar to what we did for the HMAC C-model.

Reply:
I'm probably not the right person to ask..the lowrisc guys would have a much better opinion about how best to handle this. It makes sense to move forward like for the timing being and change later if necessary though.

Reply:
If the code is properly attributed in the header, I think this should be ok for now. But let's double check with @imphil.

Also, can you add a link to the code to the header?